app/src/main/java/com/group80/uoftinder/CreateAccountView.java

```diff
@@ -0,0 +1,56 @@
+package com.group80.uoftinder;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.group80.uoftinder.create_account_use_case.CreateAccountPresenter;
+
+import java.util.HashMap;
+
+public class CreateAccountView extends AppCompatActivity {
+    HashMap<String, String> basicInfo = new HashMap<String, String>();
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.createaccountview);
+
+        Button buttonShowLoginView = findViewById(R.id.loginButton);
+        buttonShowLoginView.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                showLoginView(v);
+            }
+        });
+
+        Button enter = findViewById(R.id.accountEnter);
+        enter.setOnClickListener(new View.OnClickListener() {
+            public void onClick(View v) {
+                createAccountView();
+
+            }
+        });
+    }
+
+    public void createAccountView() {
+
+        UserAccountController control = new UserAccountController();
+        CreateAccountPresenter proceed = new CreateAccountPresenter();
+
+        String email = ((EditText) findViewById(R.id.loginEmail)).getText().toString().trim();
```

Comment:
```suggestion
        String email = ((EditText) findViewById(R.id.mail)).getText().toString().trim();
```

Note that your `loginEmail` is in your `loginView.xml`, while what you need is `email` from `createaccountview.xml` (so you will get a NullPointerException). 

My suggestion is to use `ActivityName + Function + Type` for id. For example, `createAccountViewEmailEditText` or `createAccountView_Email_EditText`. We can agree on a naming convention later, but please use something that's clear for yourself~

Reply:
See [here](https://github.com/CSC207-2022F-UofT/course-project-group-80/pull/24/files#r1027250570)

Reply:
This branch was merged early on with #24 issue to get the UI and the ability to switch between the pages. Now, the overlapping files were deleted to prevent this issue.